src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/ClaimantResponseConfirmsToProceedLiPRespondentNotificationHandler.java

```diff
@@ -132,12 +149,26 @@ RESPONDENT_NAME, getPartyNameBasedOnType(caseData.getRespondent1())
         );
     }
 
+    private boolean isFullDefenceStatesPaid(CaseData caseData) {
+        return HAS_PAID_THE_AMOUNT_CLAIMED.equals(caseData.getDefenceRouteRequired())
+            && FULL_DEFENCE.equals(caseData.getRespondent1ClaimResponseTypeForSpec());
+    }
+
     private boolean isIntermediateOrMultiClaimProceedForLipVsLR(CaseData caseData) {
         List<String> responseClaimTrack = Arrays.asList("INTERMEDIATE_CLAIM", "MULTI_CLAIM");
         return responseClaimTrack.contains(caseData.getResponseClaimTrack())
             && caseData.isLipvLROneVOne()
-            && caseData.getApplicant1ProceedWithClaim().equals(
-            YesOrNo.YES)
+            && (isFullDefenceStatesPaid(caseData)
+                || YesOrNo.YES.equals(caseData.getApplicant1ProceedWithClaim()))
             && featureToggleService.isDefendantNoCOnlineForCase(caseData);
     }
+
+    private boolean isClaimantNotProcessLipVsLRWithNoc(CaseData caseData) {
+        return caseData.isLipvLROneVOne()
+            && (
+                (isFullDefenceStatesPaid(caseData) && NO.equals(caseData.getCaseDataLiP().getApplicant1SettleClaim()))
```

Comment:
`caseData.getCaseDataLiP().getApplicant1SettleClaim()` should be YES here because if they are settling the claim this means they are not proceeding.

Reply:
solved.